lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": ["packages/*"],
+  "npmClient": "yarn",
+  "useWorkspaces": true,
+  "version": "0.0.0"
```

Comment:
i think if you put `independent`, you can version each package separately

Reply:
I was thinking they should probably be together.. since updates to the client should probably be reflected on the types

Reply:
But then you cannot apply small fixes to libraries unless ou update client, might be irrelevant, but we could pile some fixes and then release single client update with all those fixes. Irrelevant for now

Reply:
hmnm good point I think independent makes sense